android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -169,8 +169,13 @@
   public static final String KMKey_FontFiles = "files";
   public static final String KMKey_KeyboardModified = "lastModified";
   public static final String KMKey_KeyboardRTL = "rtl";
+
+  // Intend to deprecate: KMKey_CustomKeyboard is no longer used and will be deprecated in Keyman 15.0
```

Comment:
We can mark them deprecated now -- just not remove them. Deprecation is a warning that it will be removed in a future version ...

Just confirming that as this PR stands, Keyman Engine users can continue to use the API without change from 13.0 to 14.0?

Reply:
Correct. They can continue to set the property. It just won't get used.